src/extension.js

```diff
@@ -27,28 +29,30 @@ let OSKIndicator = GObject.registerClass(
 
       this.add_child(icon);
 
-      this.connect("button-press-event", function (actor, event) {
+      this.connect("button-press-event", function (_, event) {
```

Comment:
That param isn't being used, and it doesn't seem likely it would be used in the future, which is why I assigned the value to `_` instead of an actual variable. I probably should have done a better job explaining this change, but it seemed too small to invest more time on.
How about I change it to `_actor` instead? This should signal that it's not being used while maintaining the variable's name and function.